Packages/ohif-viewerbase/client/lib/classes/plugins/OHIFPlugin.js

```diff
@@ -50,7 +50,5 @@ class OHIFPlugin {
 // TODO: Move to OHIF.plugins.entryPoints?
 OHIFPlugin.entryPoints = {};
 
-export { OHIFPlugin };
-
 // TODO: Should we remove this? Authors should be able to use new 'OHIF.viewerbase.OHIFPlugin()'
 window.OHIFPlugin = OHIFPlugin;
```

Comment:
I'd think OHIF.plugins.OHIFPlugin  actually makes sense to me - not as much global namespace pollution but not so obscure as calling a method to access a variable.

Then we can also have OHIF.plugins.entryPoints.

Reply:
Makes me think we are almost to the point of writing down the API and class diagram for all the things we want to support with plugins.